### ISSUE TITLE
Frequency undersampling

### ISSUE THREAD

**Issue by pulquero** [RotorHazard/RotorHazard#72]
Add the ability for a node to scan more than one frequency. A 4-node timer can then be used as an 8-node timer at the expense of a loss of accuracy which is acceptable for practice sessions.

**Comment by HazardCreative** [comment]
The RSSI filter takes up most of the available RAM on the Arduino. To implement this in a way that's user-selectable would require a complex memory re-allocation. Those operations tend to fragment memory on small microcontrollers. I'm not saying it isn't possible, but it definitely presents some significant challenges.

**Comment by pulquero** [comment]
Yes, I see it is already quite tight on memory. Could uint8_t be used for rssi values? I think 0-256 is enough range or could do 0-512 by dropping one bit (accuracy to even numbers).

**Comment by HazardCreative** [comment]
That's an interesting idea. The highest value I've seen so far is 255 exactly, but I've only just begun collecting race history data and can't say for sure if that's enough. We could also raise the usable range of the measurement by subtracting an offset. I think dropping one bit of precision could be the best option here, though, as there's so much noise in the RSSI that there is basically no accuracy in timing from having more precision there. We could potentially process and store the timestamp with an offset as well, though reconstructing it when required would take a little bit more effort.

If we went any of those routes I'd be interested in exploring the gains from expanding the number of samples in the filter. Cleaner data makes catching passes much simpler and accurate, potentially eliminating the need for a 'backup' node entirely as in #73 ... Of course then we'd be right back where we started on memory management.

**Comment by pulquero** [comment]
Though would reduce bandwidth as less bytes required to send the values.

**Comment by pulquero** [comment]
I've seen a value higher than 255, 280.
